types/invariant.go

```diff
@@ -6,17 +6,21 @@ import "fmt"
 // The invariant returns a descriptive message about what happened
 // and a boolean indicating whether the invariant has been broken.
 // The simulator will then halt and print the logs.
+// Deprecated: to be removed in the next SDK version.
```

Comment:
```suggestion
// Deprecated: to be removed in > 0.52.
```
? next version could mean a lot of things.